pkg/project/project.go

```diff
@@ -1,7 +1,7 @@
 package project
 
 var (
-	bundleVersion        = "8.1.1-dev"
+	bundleVersion        = "8.1.1"
```

Comment:
This is a release PR which IMO should be stated in the PR title. Otherwise it's very easy to miss that.

Reply:
Can you check if the title works for you now. If not please suggest something.

Reply:
IMO this should be simply `release v8.1.1`. You can write in the description why you create the release.

Reply:
Sorted. You happy now?

Reply:
Yes.